cosmos/operators/local.py

```diff
@@ -227,7 +231,7 @@ def run_command(
                     output_encoding=self.output_encoding,
                     cwd=tmp_project_dir,
                 )
-                if is_openlineage_available:
+                if is_openlineage_enabled:
```

Comment:
@ivanstillfront Thanks for looking into this!

I'm not convinced about this change since this line aims to set Airflow inlets and outlets - unrelated to emitting open lineage events - but to support Airflow Data Aware scheduling.

This is being improved, but it will likely only be available in Cosmos 1.3 or 1.4.

Does the following comment address your original issue?
https://github.com/astronomer/astronomer-cosmos/issues/612#issuecomment-1776880897